tripal/biomaterial/__init__.py

```diff
@@ -0,0 +1,53 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+
+from tripal.client import Client
+
+logging.getLogger("requests").setLevel(logging.CRITICAL)
+log = logging.getLogger()
+
+
+class BiomaterialClient(Client):
+    """Manage Tripal organisms"""
+
+    def add_biomaterial(self, organism_id, file_path, file_type, no_wait=False):
```

Comment:
Could we move this to the `expression` client? I think it would make more sense

Reply:
Ok.
Do we keep the expression/biomaterials separated in tripaille?

Reply:
No, we merge it too there (but did you notice that the tripaille code is generated with `make rebuild`?)